CommuniTEAM/CommuniTEA
# ✨ [TEA-49]: Installed Material UI

### In this PR, I'm adding/changing the following:

- Installing and testing Material UI

### Screenshots / Code Snippets:
![image](https://github.com/CommuniTEAM/CommuniTEA/assets/57964857/972bf104-f827-4e18-930e-002d97c53df4)
![image](https://github.com/CommuniTEAM/CommuniTEA/assets/57964857/dc3dff2e-cf3f-4ca4-badd-6691b3343832)

### Please do the following to test my code:

1. `git checkout TEA-49-Install-Material-UI`
2. `git pull`
3. Start up your Docker Container 
4. Navigate to `localhost:3000`
5. You should see a MUI button, and when you click on it, a modal should pop up.

[Related Jira Ticket](https://communiteam.atlassian.net/browse/TEA-49)
